pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -8,10 +8,16 @@
 from pylint.checkers import BaseChecker
 from pylint.lint import PyLinter
 
+from homeassistant.backports.enum import StrEnum
 from homeassistant.const import Platform
 
-DEVICE_CLASS = object()
-UNDEFINED = object()
+
+class SentinelValues(StrEnum):
+    """Sentinel values"""
+
+    DEVICE_CLASS = "***DEVICE_CLASS"
+    UNDEFINED = "***UNDEFINED"
```

Comment:
That's likely what I didn't quite like with the other PR.

* `StrEnum` is basically useless here. The `repr` is printed anyway so we might as well just use a normal `Enum`. _Once the fix for the duplicate `DEVICE_CLASS` check is added, nothing will be printed._
* The name `SentinelValues` is quite verbose. It's only used in this file, so the name doesn't need to be perfect. What do you think of `Special`?

Reply:
done